mailpoet/assets/js/src/wizard/welcome_wizard_controller.jsx

```diff
@@ -150,7 +150,11 @@ function WelcomeWizardStepsController(props) {
         ) : null}
 
         {stepName === 'WizardWooCommerceStep' ? (
-          <WooCommerceController isWizardStep />
+          <WooCommerceController
+            isWizardStep
+            currentStep={step}
+            redirectToNextStep={redirect}
```

Comment:
I think you don't need to pass both `currentStep` and `redirectToNextStep`. You could use an anonymous function and save one property. But it is just a detail.
```js
redirectToNextStep={() => redirect(step)}
```

Reply:
I like your suggestion and I haven't considered it before. I just added a commit that implements your idea. Thanks.